var/spack/repos/builtin/packages/camp/package.py

```diff
@@ -39,6 +42,7 @@ def cmake_args(self):
         else:
             options.append('-DENABLE_CUDA=OFF')
 
-        options.append('-DENABLE_TESTS=ON')
+        options.append('-DENABLE_TESTS={0}'.format(
+            "On" if "tests=none" in spec else "Off"))
```

Comment:
```suggestion
            "On" if self.run_tests else "Off"))
```
Then you can get rid of the variant. Tests are controlled by `spack install --test` since they don't affect what gets installed.

Reply:
Done.